### PR TITLE
Ac nb create badges

### PR DESCRIPTION
**Pull request by itiswicked**
Associated project with the following services:

- Codeship
- Code Climate
- Hound
- Coveralls

Codeship shows that the current branch cannot be found, but all of our builds are successful.

### CODE REVIEW

File: ER_Diagram.xml [hunk 1]
```diff
@@ -0,0 +1 @@
...WvGjhTuqYpthNxZDHip9+ufWv+LuBeHhTOAbJBzaFBx7JOEzuU6lpL0DRWWbw0RBVmnW2gvqZ9hA5WI1wGWhZIwezbSJdSMJcv07CbDrxFUmYTRU7sTA8qZ3+sPKjtn4/7LjSmj+dAwFhSzigSOOKNvZvZS2t+mofa89WBn8qm+E0G093NavMsZOeXd0F2ZpB0b73wPtZLFTc+vVSG6Ba7XB4wMgTCAUZPc3u3S+J1OeWntDIw2M5pQQmBjQiWQmNSNgJjWYGNKoYnsfBpc5dwtMiYkkiFmXL8FeR/lP4/S/8tiBRCU8xqKOLCo+6dIfkPKCEZ5PlQBW5UQuqham+lKmQY7UcUesWLhhwarAfyRSPWCqey2qYbIojfJciWxXkQKua06Fn6iPXH5sV05i1nkglyk7TM/WUz7rWU+bk7cTb+d2UMj3S7+hc9HxWsvV70et5veP8TrmrAfl9SxHJ7+6xfrdnlZ4ovd6/3z1KWZ48B3gJW6TsK1l+QqajgekwJzNC/llZmvHoGLNVbyyt8xR7TkNkl1JZIkK76POlIQqZlOw4fp4ZnCbO0iWwrxBStTVf/J4pBLeC1BML8elgCqGPixwPsboX8G4mbVsk7wreBDol8M4MtocW29OxDZmOie/Mpq7nlEny1LfGjuXM/TlwpJkzQRbbEvnfAfLOZDKegy7+BIdro8b7CcDeyXqeP/Zt15q4kxlc9FeYSo/cwG3kBjXo7paqum25H9aGPS816BNdT6jmXgAfqtIFdGwDKR3ISY9JDH1dWbqa/MDsSjMkf77cyinyKPAaxTjfTV17PgeAxrz7WdIqpEsnhG5InhBim4/eRyI0bv1tEPo8x+s5nYHpkqbqlrbgOq8cruEdnJMB6LFJlQR0SNsFg4GUmYMh3437omt5gHxUMPiVVEvtdai2weBTQ+qIOj4QDOAbpqdL1da6Auy5jbzRrUSdW36FEX7kXR4davhWp4GrdUGadheLGX3r8fabrdaabT0k+utPIF0tDphtk0mDOD5fg2tLl1iDPej/kEDKYCP4BKp+k7BhtnhP7Bxk7EpIoD7rpYBXtYdBD29YnLJkIBendwtxcSOGoct3Kde/O1muwSBdWZ2FHM9MdxA90yTDxVPzNeWRNAkheLg0afuiU1eaBMmnnjgzPO7+VZJX3/1Tx/36Cw==</diagram></mxfile>
```
Comment (justinhuynh) [inline ER_Diagram.xml]: Remove this file from your git, as it's not pertinent to your actual code. Upload a screenshot to imgur and link the image in your readme.md (like you did with the badges)

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require 'coveralls'
+Coveralls.wear!('rails')
 # This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (justinhuynh) [inline spec/spec_helper.rb]: Delete comments that are automatically generated by rails

**Comment by itiswicked** [comment]
Updates:

Code comments deleted from

**Comment by itiswicked** [comment]
Changes:

get rid of rails generated code comments.
deleted ER diagram.

**Pull request merged**